### ISSUE TITLE
Copied code from my steps extension

### ISSUE THREAD

**Issue by aramvisser** [Wikiki/bulma-steps#4]
Hi @wikiki,

There are some issues with your steps extension code. I noticed that you changed the look for your steps extension since I made [my extension](https://github.com/aramvisser/bulma-steps). It now more closely resembles mine.

I can also clearly tell you directly copied some code from me. Your [`steps-size` sass mixin](https://github.com/Wikiki/bulma-steps/commit/b7e8039c1f7d7fcf1e79b6f5b24887d1faa65486#diff-d20ceb9652c26e5d67b2f3c34406053bR8) is very similar with the _exact_ same comment above it.

Your [steps examples](https://wikiki.github.io/bulma-extensions/steps) in the documentation are also taken directly from me (you only use 4 steps, but your final step still says step 5).

You also added very similar css classes that I used, like `step-marker` and `step-content`.

Can you either rewrite all those parts to be your own or give me credit to say where you got it from?

**Comment by Wikiki** [comment]
Hi @aramvisser,

You're right, I took you copied your example text for my documentation -> It as been changed now.

I also renamed my selector from 'tag' to 'marker' because I found your choose for the name is more relevant that my old one: I'll give you credit for that.

I also took your mixin declaration you're right but not the idea and not the way to implement it (only the declaration).

I've added a credit to you on my documentation page and in the README.

Kindly regards,
